### DISCUSSION TITLE
NextcloudPi - fail2ban read error

### DISCUSSION THREAD

**Discussion by nicedevil007** [tteck/Proxmox#1000] [Help Needed]
Hi guys,

I just installed NextcloudPi and I'm getting an error: `fail2ban read error`
![image](https://user-images.githubusercontent.com/17103076/215580716-6fb67efe-7020-4e09-b40a-86915b87b8f4.png)

This showing up during the first run setup and I chose Yes on the question about the external access to my Nextcloud.

**Comment by tteck** [comment]
Don't know how that works, but I'd guess you'd skip that and use Nginx Proxy Manager ect.. to reach the outside world.

**Reply by nicedevil007** [reply]
Sure already existing, but I guess that it want to configure fail2ban to if you choose this option.
I installed it manually now and trying again

**Reply by Ckephar** [reply]
I also had the same issue. Did installing manually resolve this?

**Reply by nicedevil007** [reply]
unforunately nope
